src/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -426,7 +428,8 @@ export class MongoEntityConverter {
 
   public convertMedia(mongoMedia: MongoMedia): Media {
     return {
-      id: mongoMedia.mediaId,
+      id: mongoMedia.id,
```

Comment:
You made `MongoMedia` extend `MongoId` so this should be `id: mongoMedia._id`, because `MongoMedia.id` shouldn't exist when you are extending `MongoId`.

Reply:
Changed!